doc/self-hosted/cros-install.md

```diff
@@ -1,20 +1,31 @@
-# Installng code-server in your ChromiumOS/ChromeOS/CloudReady machine
+# Installng code-server on a ChromeOS/CloudReady machine
 
-This guide will show you how to install code-server into your CrOS machine.
+This guide will show you how to install code-server on your CrOS machine.
 
 ## Using Crostini
 
-One of the easier ways to run code-server is via [Crostini](https://www.aboutchromebooks.com/tag/project-crostini/), the Linux apps support feature in CrOS. Make sure you have enough RAM, HDD space and your CPU has VT-x/ AMD-V support. If your chromebook has this, then you are qualified to use Crostini.
+One of the easier ways to run code-server is via [Crostini](crostini), the Linux
```

Comment:
there's a large gap in here, might be GitHub wrapping but this is supposed to be one line.

Reply:
Not sure what caused it, seems like a GitHub rendering issue.